.github/workflows/release-cosmos-sdk.yaml

```diff
@@ -0,0 +1,328 @@
+name: Nightly builds of SimApp (Cosmos-SDK)
+on:
+  schedule:
+    - cron: 0 0 * * * # Every day at 0:00 UTC
+  workflow_dispatch:
+
```

Comment:
short comment of the all the things the action does would be useful. Allows people to avoid reading through all the steps

Reply:
added !